packages/did-auth-siop-op-authenticator/__tests__/shared/didAuthSiopOpAuthenticatorAgentLogic.ts

```diff
@@ -311,14 +327,27 @@ export default (testContext: {
     })
 
     it('should get authentication details with multiple credentials', async () => {
-      const result = await agent.getSiopAuthenticationRequestDetails({
+      let authorizationRequestArgs: IGetSiopAuthorizationRequestDetailsArgs = {
         sessionId,
-        verifiedAuthenticationRequest: {
-          ...createAuthenticationResponseMockedResult,
+        verifiedAuthorizationRequest: {
+          ...createAuthorizationResponseMockedResult,
           presentationDefinitions: pdMultiple,
+          authorizationRequest: {} as AuthorizationRequest,
+          versions: [],
+          authorizationRequestPayload: {
+
+          },
+          payload: {}
         },
         verifiableCredentials: vcs,
-      })
+        presentationSignCallback: async (args) => {return 'vc 202212272109' as W3CVerifiablePresentation},
```

Comment:
Please do something proper with vc-js/jose  for instance. This certainly isn't returning a VP

Reply:
fixed